.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -1,37 +1,83 @@
 name: Parser compilation and query file check
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - 'master'
+
+# Cancel any in-progress CI runs for a PR if it is updated
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
 
 jobs:
-  check_compilation_unix_like:
+  check_compilation_unix_latest:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        cc: [ gcc ]
+        nvim_tag: [ nightly ]
```

Comment:
This shouldn't be nightly. Our main CI is latest stable on Ubuntu. Nightly is not always available for download and is therefore allowed to fail

Reply:
It is also very annoying to change the required builds. You won't be able to merge this and I need to change it manually after force-merging this PR

Reply:
OK, so you _do_ want me to drop the fast draft CI -- you just have to say so!